kolibri/plugins/user_profile/tasks.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management import call_command
+from rest_framework import serializers
+
+from kolibri.core.auth.models import FacilityUser
+from kolibri.core.auth.tasks import PeerImportSingleSyncJobValidator
+from kolibri.core.auth.utils.migrate import merge_users
+from kolibri.core.serializers import HexOnlyUUIDField
+from kolibri.core.tasks.decorators import register_task
+from kolibri.core.tasks.job import Priority
+from kolibri.core.tasks.permissions import IsFacilityAdmin
+from kolibri.core.tasks.permissions import IsSelf
+from kolibri.core.tasks.permissions import IsSuperAdmin
+from kolibri.core.tasks.permissions import PermissionsFromAny
+
+
+class MergeUserValidator(PeerImportSingleSyncJobValidator):
+    local_user_id = HexOnlyUUIDField()
```

Comment:
This could also be a `PrimaryKeyRelatedField` with the queryset as the `FacilityUser.objects.all()`

Reply:
good catch, I'm adding this modification in the PR to close #9355